### ISSUE TITLE
Support for swift 3?

### ISSUE THREAD

**Issue by learnwithgabbar** [sunshinejr/RxPermission#8]

**Comment by sunshinejr** [comment]
Yeah, will upgrade as soon as possible with a new version candidate :)

**Comment by roshanman** [comment]
I also need this for swift3, thank you.

**Comment by sunshinejr** [comment]
We have to wait for RxSwift & Permission to officially support Swift 3.0, before we can support it here unfortunately :<

**Comment by learnwithgabbar** [comment]
Rx swift is already upgraded into swift 3.0

**Comment by sunshinejr** [comment]
Yeah but Permission is not.

**Comment by sunshinejr** [comment]
Hey guys, @roshanman, @techmehra, RxPermission for Swift 3.0 is out, in the beta stage as long as RxSwift is in beta and Permission isn't out. If you guys encounter any troubles, please make another issue! Thanks! 🙇
